aca_preview/tests/test_checks.py

```diff
@@ -34,66 +34,56 @@ def test_check_P2():
     assert 'less than 3.0 for ER' in msg['text']
 
 
-def test_enough_guide_er():
-    """Test the check that an ER has enough guide stars"""
-    stars = StarsTable.empty()
-    stars.add_fake_constellation(n_stars=5, mag=8.0)
-    aca = get_aca_catalog(**mod_std_info(n_fid=0, n_guide=8), stars=stars, dark=DARK40,
-                          raise_exc=True)
-    ACAReviewTable.add_review_methods(aca, obsid=50000)
-    aca.check_enough_guide_for_ers()
-    assert len(aca.messages) == 1
-    msg = aca.messages[0]
-    assert msg['category'] == 'critical'
-    assert 'ER guide stars' in msg['text']
+def test_guide_count_er():
+    """Test the check that an ER has enough fractional guide stars by guide_count"""
 
+    # This configuration should have not enough bright stars
     stars = StarsTable.empty()
-    stars.add_fake_constellation(n_stars=6, mag=8.0)
+    stars.add_fake_constellation(n_stars=8, mag=9.5)
     aca = get_aca_catalog(**mod_std_info(n_fid=0, n_guide=8), stars=stars, dark=DARK40,
                           raise_exc=True)
     ACAReviewTable.add_review_methods(aca, obsid=50000)
-    aca.check_enough_guide_for_ers()
+    aca.check_guide_count()
     assert len(aca.messages) == 1
     msg = aca.messages[0]
-    assert msg['category'] == 'warning'
-    assert 'ER guide stars' in msg['text']
-
+    assert msg['category'] == 'critical'
+    assert 'ER guide count' in msg['text']
 
-def test_guide_fraction_er():
-    """Test the check that an OR or ER has enough fractional guide stars by guide_count"""
+    # This configuration should have not enough stars overall
     stars = StarsTable.empty()
-    stars.add_fake_constellation(n_stars=8, mag=9.5)
+    stars.add_fake_constellation(n_stars=3, mag=[8.5, 8.5, 8.5])
     aca = get_aca_catalog(**mod_std_info(n_fid=0, n_guide=8), stars=stars, dark=DARK40,
                           raise_exc=True)
     ACAReviewTable.add_review_methods(aca, obsid=50000)
-    aca.check_guide_fraction()
+    aca.check_guide_count()
     assert len(aca.messages) == 1
     msg = aca.messages[0]
     assert msg['category'] == 'critical'
-    assert 'ER guide count' in msg['text']
+    assert 'Guide count' in msg['text']
```

Comment:
These messages should be very stable, so test the entire thing (including the output numbers).  As written it is difficult to tell what's going on.

Reply:
Fixed.